### PR TITLE
PARQUET-2483: Update references to parquet-mr in Readmes

### PR DESCRIPTION
**Pull request by emkornfield**
Make sure you have checked _all_ steps below.

### Jira

- [x ] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references
  them in the PR title. For example, "PARQUET-1234: My Parquet PR"
    - https://issues.apache.org/jira/browse/PARQUET-XXX
    - In case you are adding a dependency, check if the license complies with
      the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [x ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x ] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines
  from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    1. Subject is limited to 50 characters (not including Jira issue reference)
    1. Subject does not end with a period
    1. Subject uses the imperative mood ("add", not "adding")
    1. Body wraps at 72 characters
    1. Body explains "what" and "why", not "how"

### Style
- [x ] My contribution adheres to the code style guidelines and Spotless passes.
    - To apply the necessary changes, run `mvn spotless:apply -Pvector-plugins`

### Documentation

- [x ] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
   ~ under the License.
   -->
 
-Parquet MR [![Build Status](https://github.com/apache/parquet-mr/workflows/Test/badge.svg)](https://github.com/apache/parquet-mr/actions)
+Parquet MR [![Build Status](https://github.com/apache/parquet-java/workflows/Test/badge.svg)](https://github.com/apache/parquet-mr/actions)
```
Comment (wgtmac) [inline README.md:20]:
```suggestion
Parquet Java [![Build Status](https://github.com/apache/parquet-java/workflows/Test/badge.svg)](https://github.com/apache/parquet-mr/actions)
```

Comment (wgtmac) [reply README.md:20]: Should we also change this?

Comment (emkornfield) [reply README.md]: I think I did this?

**Review by wgtmac — APPROVED** [review]

**Review by Fokko — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ You can find additional details about the format and intended use cases in our [
 
 ## Building
 
-Parquet-MR uses Maven to build and depends on the thrift compiler (protoc is now managed by maven plugin).
+Parquet-java uses Maven to build and depends on the thrift compiler (protoc is now managed by maven plugin).
```
Comment (Fokko) [inline README.md:39]: To be more consistent with the casing:
```suggestion
Parquet-Java uses Maven to build and depends on the thrift compiler (protoc is now managed by maven plugin).
```

**Comment by emkornfield** [comment]
If no objections I can Squash merge this and resolve the jira by hand?

**Pull request closed without merging**